lib/prelude.dx

```diff
@@ -1866,6 +1866,7 @@ Not to be confused with a partially applied function
 
 '### Error throwing
 
+@noinline
```

Comment:
It would be sufficient to mark `print`, wouldn't it? `error` is not much larger than that.

Reply:
Done.